src/main/java/org/sciborgs1155/robot/pivot/Pivot.java

```diff
@@ -0,0 +1,160 @@
+package org.sciborgs1155.robot.pivot;
+
+import static edu.wpi.first.units.Units.*;
+import static org.sciborgs1155.robot.pivot.PivotConstants.*;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Direction;
+import java.util.function.Supplier;
+import monologue.Annotations.Log;
+import monologue.Logged;
+import org.sciborgs1155.robot.Constants;
+import org.sciborgs1155.robot.Robot;
+
+public class Pivot extends SubsystemBase implements AutoCloseable, Logged {
+  @Log.NT private final PivotIO pivot;
+  private final SysIdRoutine sysIdRoutine; //sysIdRoutineoogabooga
+
+  // pivot control
+  @Log.NT 
+  final ProfiledPIDController pivotPID =
+      new ProfiledPIDController(
+          PivotConstants.kP,
```

Comment:
you static import PivotConstants at the top of the file, so you can directly access the variables without prefixing `PivotConstants`

Reply:
Would like to keep it consistent with how we access constants in ClimberConstants